### DISCUSSION TITLE
Have a q and a area

### DISCUSSION THREAD

**Discussion by WhyIsEmerald** [dailydotdev/daily#1179] [Feature Requests]
It would be nice if you could have a q and a area as most daily.dev is a casual place to socialize with other developers it could serve as a nicer stack overflow

**Comment by idoshamun** [comment]
Hi @emerald-developer,

That's a great suggestion. We see squads turn into something like this 🤞

**Reply by WhyIsEmerald** [reply]
The problem with it is that it requires a post with thumbnail etc not just an area to discuss.

**Reply by idoshamun** [reply]
You can write posts with no thumbnail. Title is the only required field which is great for a question, right?

**Reply by WhyIsEmerald** [reply]
but it still appears as a card would it be nice to have something like this discussions page n which there is just text.

**Reply by idoshamun** [reply]
I see. Something like a list view?

**Reply by WhyIsEmerald** [reply]
yes and something separate from the news section so readers can read while people who want to answer question and earn reputation can go to a q&a section.

**Reply by WhyIsEmerald** [reply]
this can help get the website traction in searches so more question more likely someone might find it

**Reply by omBratteng** [reply]
@idoshamun maybe in some squads, it would make more sense to use the list view. Kinda like how Discord have special channels that can act like "forums".
